discovery_engine_core/ai/ai/src/ranker/system.rs

```diff
@@ -159,6 +161,9 @@ impl Ranker {
                 .log_negative_user_reaction(&mut self.state.user_interests.negative, embedding),
             UserFeedback::NotGiven => {}
         }
+        if tracing::enabled!(Level::DEBUG) {
```

Comment:
the docs of `enabled!` say that this macro is meant to be used if the emitted event contains expensive computations. since we only display fields (in this case and the other cases where `enabled!` was used), we can likely remove this and call the event macro directly.